x-pack/plugins/observability/public/pages/slos/components/grouped_slos/group_list_view.tsx

```diff
@@ -18,13 +18,29 @@ interface Props {
   sloView: string;
   sort: string;
   direction: SortDirection;
+  groupBy: string;
 }
 
-export function GroupListView({ isCompact, group, kqlQuery, sloView, sort, direction }: Props) {
-  const query = kqlQuery ? `"slo.tags": ${group} and ${kqlQuery}` : `"slo.tags": ${group}`;
+export function GroupListView({
+  isCompact,
+  group,
+  kqlQuery,
+  sloView,
+  sort,
+  direction,
+  groupBy,
+}: Props) {
+  let query = '';
```

Comment:
Need to think more about this one as part of https://github.com/elastic/kibana/issues/175533. We need to make sure that Grouping works well with Unified Search.